KubeArmor/core/containerdHandler.go

```diff
@@ -7,11 +7,14 @@ package core
 import (
 	"context"
 	"fmt"
+	"github.com/containerd/typeurl/v2"
 	"os"
 	"strconv"
 	"strings"
 	"time"
 
+	"golang.org/x/exp/slices"
```

Comment:
slices package is stable now, we don't need the exp import now. https://pkg.go.dev/slices

Reply:
It moved out of experimental in go1.21. We'll need to update go version.

Reply:
IMO we should do it ASAP. Dev environments which use go 1.21 don't work well with projects using go1.20.
Running things like go mod tidy adds weird directives which don't work with older go versions. Even gopls doesn't offer any completion/documentation for older projects.

Reply:
I think we should have a new PR for updating go versions.

Reply:
Can you open an issue, we can resolve the comment then